lib/helpbase.gi

```diff
@@ -1013,13 +1013,74 @@ InstallGlobalFunction(HELP_GET_MATCHES, function( books, topic, frombegin )
   return [exact, match];
 end);
 
+if not IsBound( InitialSubstringUTF8String ) then
+  InitialSubstringUTF8String:= "dummy";
+fi;
+if not IsBound( LETTERS ) then
+  LETTERS:= "dummy";
+fi;
+if not IsBound( WidthUTF8String ) then
+  WidthUTF8String:= "dummy";
+fi;
+
+BindGlobal( "InitialSubstringUTF8Text", function( str, cols )
+    local esc, parts, len, res, j, pos, word, w;
+
+    # Collect printable parts up to visible length 'cols',
+    # and all "escape sequences" in 'str'
+    # (sequences starting with ESC and stopping with the first letter
+    # afterwards).
```

Comment:
Is this a full description of what this function does? Then perhaps it could be moved to be just before the function, then this would be clearer, IMHO